app/controllers/concerns/sufia/controller.rb

```diff
@@ -28,24 +28,24 @@ def render_single_use_error(exception)
   end
 
   def notifications_number
+    # These instance variables must appear above the return statements
     @notify_number = 0
     @batches = []
     return if action_name == "index" && controller_name == "mailbox"
-    if user_signed_in?
-      @notify_number = current_user.mailbox.inbox(unread: true).count
-      @batches = current_user.mailbox.inbox.map { |msg| msg.last_message.body[/<span id="(.*)"><a (href=|data-content=|rel=)(.*)/,1] }.select{ |val| !val.blank? }
-    end
+    return unless user_signed_in?
+    @notify_number = current_user.mailbox.inbox(unread: true).count
+    @batches = current_user.mailbox.inbox.map { |msg| msg.last_message.body[/<span id="(.*)"><a (href=|data-content=|rel=)(.*)/, 1] }.select { |val| !val.blank? }
   end
 
   # Override Devise method to redirect to dashboard after signing in
-  def after_sign_in_path_for(resource)
+  def after_sign_in_path_for(_resource)
```

Comment:
Yep!  Convention to mark unused variables.  (I doubt we'll do this very often -- usually we only declare variables we use.)